src/projectscene/internal/projectsceneuiactions.cpp

```
@@ -195,6 +195,30 @@ static UiActionList STATIC_ACTIONS = {
+             muse::shortcuts::CTX_PROJECT_OPENED,
+             TranslatableString("action", "Extend selection right"),
+             TranslatableString("action", "Extend selection right")
+             ),
+    UiAction("sel-cntr-left",
+             au::context::UiCtxProjectOpened,
+             muse::shortcuts::CTX_PROJECT_OPENED,
+             TranslatableString("action", "Contract selection from left"),
+             TranslatableString("action", "Contract selection from left")
+             ),
+    UiAction("sel-cntr-right",
+             au::context::UiCtxProjectOpened,
```

Cppcheck (2.20.0)

[style] The function 'fromProjectFile' is never used.

(unusedFunction)